schema/query/fuzz.go

```diff
@@ -0,0 +1,51 @@
+// +build gofuzz
+
+package query
```

Comment:
IF only access to the public interface is needed, switch to `package query_test`.